tests/unit/components/common/StatusList.spec.ts

```diff
@@ -29,4 +30,16 @@ describe('StatusList.vue', () => {
     })
     expect(wrapper.find('[data-test="select-status"]').exists()).toBeTruthy()
   })
+
+  it('does not render Refund Rejected option', () => {
+    const wrapper = shallowMount(StatusList, {
+      localVue,
+      vuetify,
+      propsData: {
+        value: 'ACTIVE'
+      }
+    })
+    // Check if the "Refund Rejected" option is not in the status list
+    expect(wrapper.text()).not.toContain('Refund Rejected')
```

Comment:
You'll need to update the`routingSlipStatusListMock` above so that it actually includes the refund rejected code for this to be a valid test. It would also be good to move the wrapper into the beforeEach block in both tests so that there's less code duplication

Reply:
Also please use the enums for the statuses/status labels in the mock and text comparison

Reply:
Actually I was just looking at how this test is setup and you'll need to add a new mock over CodesService.getCodes (mock the api call) -- we have examples of this elsewhere, but let me know if you want help.

Also test wrapper.text() won't work since it is a drop down select so you'll have to either simulate the dropdown opening first or test the data being passed to the v-select